python/lsst/summit/utils/quickLook.py

```diff
@@ -89,22 +89,32 @@ class QuickLookIsrTask(pipeBase.PipelineTask):
     ConfigClass = QuickLookIsrTaskConfig
     _DefaultName = "quickLook"
 
-    def __init__(self, **kwargs):
+    def __init__(self, isrTask=IsrTask, **kwargs):
         super().__init__(**kwargs)
+        # Pass in IsrTask so that we can modify it slightly for unit tests.
+        self.isrTask = IsrTask
```

Comment:
This took me longer to understand than I care to admit, so I think a slightly more thorough comment might be useful here. Perhaps something along the lines of
```
        # Pass in IsrTask so that we can modify it slightly for unit tests.
        # Note that this is not an instance of the IsrTask class, but the class
        # itself, which is then instantiated later on, in the run() method,
        # with the dynamically generated config.
```